frappe/core/doctype/docfield/docfield.json

```diff
@@ -192,7 +192,7 @@
   },
   {
    "default": "0",
-   "depends_on": "eval:['Data', 'Select', 'Text', 'Small Text', 'Text Editor'].includes(doc.fieldtype)",
+   "depends_on": "eval:['Data', 'Select', 'Text', 'Small Text', 'Text Editor', 'Link'].includes(doc.fieldtype)",
```

Comment:
```suggestion
   "depends_on": "eval:['Data', 'Select', 'Text', 'Small Text', 'Text Editor', 'Link', 'Dynamic Link'].includes(doc.fieldtype)",
```

Should we include dynamic link as well? Or any other ideas about how we could handle this?